modules/calib/src/calibration.cpp

```diff
@@ -916,13 +895,17 @@ static double stereoCalibrateImpl(
     Mat J_LRBuf( maxPoints*2, 6, CV_64F );
     Mat JiBuf( maxPoints*2, NINTRINSIC, CV_64F, Scalar(0) );
 
-    auto lmcallback = [&](InputOutputArray _param, OutputArray JtErr_, OutputArray JtJ_, double& errnorm)
+    auto lmcallback = [errBuf, JeBuf, J_LRBuf, JiBuf, recomputeIntrinsics, nimages, flags, aspectRatio, &A, &dk, NINTRINSIC,
```

Comment:
MSVS 2015 build is broken: http://pullrequest.opencv.org/buildbot/builders/5_x-win64-vc14/builds/100000

There was a failure in precommit: http://pullrequest.opencv.org/buildbot/builders/precommit_windows64/builds/100100
but nobody looks on that.
Also this builder was not triggered before merge after commits squashing.